packages/lesswrong/server/rateLimitUtils.ts

```diff
@@ -136,12 +142,12 @@ async function getCommentsOnOthersPosts(comments: Array<DbComment>, userId: stri
   return commentsOnNonauthorPosts
 }
 
-async function getCommentRateLimitInfos({commentsInTimeframe, user, modRateLimitHours, modPostSpecificRateLimitHours, postId, userCommentRateLimit, recentKarmaInfo, context}: {
+async function getCommentRateLimitInfos({commentsInTimeframe, user, modRateLimitHours, modPostSpecificRateLimitHours, postId, manualCommentRateLimit, recentKarmaInfo, context}: {
```

Comment:
This isn't your fault, but this function name is a rough one, and deserves a docstring on the function, and on the fields of the return type.

Reply:
Would like to avoid doing that now, on the principle of "Just because I am making some code marginally better, shouldn't put me on the hook for fixing everything that could be fixed about that code"